src/GrassPokemon.java

```diff
@@ -0,0 +1,41 @@
+public class GrassPokemon extends PokemonSuper {
+
+    private int grassPower;
+    private int grassSuperPower;
+
+    public GrassPokemon(String name, int level, int hp, int xp, int grassPower, int grassSuperPower) {
+        super(name, level, hp, xp);
+        this.grassPower = grassPower;
+        this.grassSuperPower = grassSuperPower;
+    }
+
+    public void leafStorm(){
+        System.out.println("Your Storm Leaf attack did 100 damage");
```

Comment:
Misschien een leuke toevoeging is om ook de vijand Pokemon hier als parameter te vragen en van die vijand ook echt 100 hp af te trekken.